schema.sql

```diff
@@ -28,4 +28,34 @@ CREATE TABLE species (
 
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(species_id);
-ALTER TABLE animals ADD COLUMN owners_id INT REFERENCES owners(owners_id);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN owners_id INT REFERENCES owners(owners_id);
+
+CREATE TABLE vets (
+    vet_id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100) NOT NULL,
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY(vet_id)
+);
+
+CREATE TABLE specializations (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    vet_id INT,
+    species_id INT,
+    PRIMARY KEY(id)
+);
+
+ALTER TABLE specializations ADD FOREIGN KEY (vet_id) REFERENCES vets(vet_id);
+ALTER TABLE specializations ADD FOREIGN KEY (species_id) REFERENCES species(species_id);
+
+CREATE TABLE visits (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    vet_id INT,
+    animal_id INT,
+    PRIMARY KEY(id)
+);
```

Comment:
Great job @OyePriscilla 🤝

- [ ] Kindly add `visit_date` to your visits table since this visits table should keep track of the date of the visit.

Reply:
Hello @1GORDON, thanks for reviewing my work! I added the day of visit to the table, perhaps you didn't noticed. The screenshot below shows my schema.sql.
![day of visit added](https://user-images.githubusercontent.com/103386183/190624449-db6ef1a2-267e-4ad0-b0d1-a9b37243de07.png)

Hope to hear from you soon.
Thanks.